clusterloader2/GETTING_STARTED.md

```diff
@@ -0,0 +1,227 @@
+# ClusterLoader
+
+In this tutorial, we will:
+- Set-up perf-tests repository for local development
+- Create single node cluster using [Kind]
+- Implement a simple Clusterloader2 test and run it
+- Run load test on 100 nodes cluster
+
+## Clone perf-test repository
+
+Start with cloning perf-test repository:
+```bash
+git clone git@github.com:kubernetes/perf-tests.git
+cd perf-tests
+```
+
+## Install GVM
+Follow instructions on [GVM install].
+Install golang with specific version (1.15.12 was tested in this tutorial):
+```bash
+gvm install go1.15.12
+gvm use go1.15.12
+```
+Next, add perf-test repository to GOPATH:
+
+```bash
+gvm linkthis k8s.io/perf-tests
+```
+
+## Create cluster using kind
+Follow kind installation guide [Kind install]
+
+Create cluster v1.21.1 with one master and one node:
+```bash
+kind create cluster --image=kindest/node:v1.21.1 --name=test-cluster --wait=5m
+```
+
+Check that you can connect to cluster:
+```bash
+kubectl get nodes
+```
+
+This command generates cluster access credentials, which are stored in `${HOME}/.kube/config` with context name test-cluster.
```

Comment:
Move above the get nodes (get nodes calls doesn't generate credentials).

Reply:
Done.